### ISSUE TITLE
训练分类和自表达矩阵分开训练的吗？

### ISSUE THREAD

**Issue by dugzzuli** [buschju/sscn#1]
您好。
看了本仓库的代码之后，有一点疑惑。这个分类步骤和子表达矩阵是分成两步骤训练？

`# Forward pass
    Z = self.encoder(X)
    H = self.se_encoder(Z).detach() #这个地方断开了，而且并没有优化子表达损失和重构损失了。

    for _ in range(num_iterations_per_batch):
        outputs = self.classifier.get_outputs(H)
        loss_assignment = get_loss_supervised(outputs=outputs,
                                              y=pseudo_labels,
                                              )

        # Backward pass
        optimizer.zero_grad()
        loss_assignment.backward()
        optimizer.step()`
